src/Microsoft.PowerShell.Commands.Management/commands/management/GetChildrenCommand.cs

```diff
@@ -172,6 +172,25 @@ public uint Depth
         }
 
         /// <summary>
+        /// Gets or sets whether symbolic links are to be followed when recursing.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter FollowSymlink
```

Comment:
So then `-FollowSymlink` should not imply `-Recurse`? It has meaning only if `-Recurse` is also explicitly given?